src/main/java/com/cx/restclient/ast/dto/sca/AstScaConfig.java

```diff
@@ -17,8 +17,9 @@
     private String tenant;
     private String webAppUrl;
     private boolean includeSources;
```

Comment:
Let's add a proper comment for the includeSources field (it has already caused confusion). Or rename it - but first ask the Plugins team.

Reply:
true: upload all sources for scan
false: only upload manifest and fingerprints for scan